coordinator/src/main/java/org/apache/uniffle/coordinator/CoordinatorServer.java

```diff
@@ -195,6 +197,12 @@ private void registerRESTAPI() throws Exception {
     jettyServer.addServlet(
         new NodesServlet(this),
         "/api/server/nodes");
+    jettyServer.addServlet(
+        new LostedNodesServlet(this),
+        "/api/server/lostedNodes");
+    jettyServer.addServlet(
+        new UnhealthyNodesServlet(this),
+        "/api/server/unhealthyNodes");
```

Comment:
We can pass status in `NodesServlet`, so we don't need add more servlets

Reply:
ok,I'll change it.